src/bp-core/js/vendor/livestamp.js

```diff
@@ -1,129 +1,224 @@
+		}
+	};
+
+	bbHumanizeFormat = function ( timestamp ) {
+		var currentDate = moment();
+		var fromDate 	= moment( timestamp ).utc();
+
+		const duration = moment.duration( currentDate.diff( fromDate ) );
+		const since    = duration.asSeconds();
+		var output     = '';
+
+		if ( 0 > since ) {
+			output = bb_livestamp.unknown_text;
+		} else {
+			var count   = 0;
+			var seconds = 0;
+
+			for ( var i = 0, j = bb_livestamp.chunks.length; i < j; ++i ) {
+				seconds = bb_livestamp.chunks[ i ];
+
+				// Finding the biggest chunk (if the chunk fits, break).
+				count = Math.floor( since / seconds );
+				if ( 0 !== count ) {
+					break;
+				}
+			}
+
+			var floor_count = Math.floor( count );
+
+			if ( 'undefined' === typeof bb_livestamp.chunks[ i ] ) {
+				output = bb_livestamp.right_now_text;
+			} else {
+
+				switch ( seconds ) {
+					case parseInt( bb_livestamp.year_in_seconds ):
+						output = ( count < 2 ) ? bb_livestamp.year_text : bbConcateString( count, bb_livestamp.years_text );
+						break;
+					case ( parseInt( bb_livestamp.year_in_seconds ) / 6 ):
+						var month_seconds = Math.floor( since / ( 30 * parseInt( bb_livestamp.day_in_seconds ) ) );
+
+						output = ( month_seconds < 2 ) ? bb_livestamp.month_text : bbConcateString( month_seconds, bb_livestamp.months_text );
+						break;
+					case ( 30 * parseInt( bb_livestamp.day_in_seconds ) ):
+						var week_seconds = Math.floor( since / parseInt( bb_livestamp.week_in_seconds ) );
+
+						if ( count < 2 ) {
+							output = ( week_seconds < 2 ) ? bb_livestamp.week_text : bbConcateString( week_seconds, bb_livestamp.weeks_text );
+						} else {
+							output = ( count < 2 ) ? bb_livestamp.month_text : bbConcateString( floor_count, bb_livestamp.months_text );
+						}
+						break;
+					case parseInt( bb_livestamp.week_in_seconds ):
+						output = ( count < 2 ) ? bb_livestamp.week_text : bbConcateString( floor_count, bb_livestamp.weeks_text );
+						break;
+					case parseInt( bb_livestamp.day_in_seconds ):
+						output = ( count < 2 ) ? bb_livestamp.day_text : bbConcateString( floor_count, bb_livestamp.days_text );
+						break;
+					case parseInt( bb_livestamp.hour_in_seconds ):
+						output = ( count < 2 ) ? bb_livestamp.hour_text : bbConcateString( floor_count, bb_livestamp.hours_text );
+						break;
+					case parseInt( bb_livestamp.minute_in_seconds ):
+						output = ( count < 2 ) ? bb_livestamp.minute_text : bbConcateString( floor_count, bb_livestamp.minutes_text );
+						break;
+					default:
+						output = ( count < 2 ) ? bb_livestamp.second_text : bbConcateString( floor_count, bb_livestamp.seconds_text );
+				}
+
+				// No output, so happened right now.
+				if ( ! parseInt( count ) ) {
+					output = bb_livestamp.right_now_text;
+				}
+			}
+		}
+
+		output = bb_livestamp.ago_text.replace( "%s", output );
+
+		return output;
+	};
+
+	bbConcateString = function ( string1, string2 ) {
+		return string1 + ' ' + string2;
+	};
```

Comment:
Please remove this additional function from vendor JS and revert it as it is.
Move this function to `src/bp-templates/bp-nouveau/js/buddypress-nouveau.js`
and use code snippet there to made it working.
```
$('[data-livestamp]').on('change.livestamp', function(event, from, to) {
				event.preventDefault();
				var timestamp = event.timeStamp;
				var to = bbHumanizeFormat( timestamp );


				if ( from != to ) {
					var e = $.Event( 'change.livestamp' );
					$this.trigger( e, [from, to] );
					if ( ! e.isDefaultPrevented() ) {
						$this.html( to );
					}
				}
			}).livestamp();